scrapers/FRA/index.js

```diff
@@ -43,7 +45,8 @@ const scraper = {
       }
     }
     states.push(transform.sumData(states));
-    return states;
```

Comment:
wait, poor FRA

Reply:
Yeah, just making sure that my code works as intended